tests/framework/PostProcessors/ParetoFrontierPostProcessor/test_paretoFrontierWithBothLimits.xml

```diff
@@ -0,0 +1,63 @@
+<Simulation verbosity="debug">
+  <TestInfo>
+    <name>framework/PostProcessors/ParetoFrontierPostProcessor.ParetoFrontier</name>
```

Comment:
change ```ParetoFrontier``` to ``ParetoFrontierWithBothLimits``

Reply:
edited